roles/uperf/templates/workload.yml.j2

```diff
@@ -195,6 +195,18 @@ items:
               configMap:
                 name: uperf-test-{{ trunc_uuid }}
           restartPolicy: OnFailure
+{% if workload_args.tuned is sameas true %}
+          serviceAccountName: benchmark-operator
+          securityContext:
+            privileged: true
+            sysctls:
+            - name: net.ipv4.ip_local_port_range
+              value: 1024 65535
+            - name: net.ipv4.tcp_tw_reuse
```

Comment:
I don't think we want this based on recent comments from Tim?

Reply:
After testing both tuning profiles we'll keep the original one.